packages/captp/lib/index.js

```diff
@@ -0,0 +1,6 @@
+import Nat from '@agoric/nat';
+
+export * from '@agoric/marshal';
+
+export * from './captp';
+export { Nat };
```

Comment:
Why are we re-exporting `Nat`?  That seems a little weird (not objecting per se, it just seems weird).

Reply:
It's a backward-compatibility issue more than anything.  I'll probably remove the `index.js` in a future PR when I'm sure we're not using it anymore.